frontend/src/Frontend/UI/Transfer.hs

```diff
@@ -835,7 +834,8 @@ crossChainTransferAndStatus model netInfo ti cmd mdestGP destSigners = Workflow
             el "p" $ text $ "Cross chain transfer initiated on chain " <> _chainId fromChain
 
           keys <- sample $ current $ model ^. wallet_keys
-          runUnfinishedCrossChainTransfer logL netInfo keys fromChain toChain mdestGP rk
+          toChainMeta <- sample $ flip transferMetaToPublicMeta toChain <$> current meta
```

Comment:
Instead of passing this in as a Dynamic and then calling sample on it, can we just pass in a pure TransferMeta?

Reply:
Well, it's not pure. The GasLimit and other metadata comes from the input form, and can be set to anything the user wants

Reply:
Oh right, I missed that bit.